package.json

```diff
@@ -27,6 +28,7 @@
   },
   "dependencies": {
     "demux-js": "^1.0.2",
+    "mongodb": "^3.1.3",
```

Comment:
Now we are adding `mongodb` dependencies to every package that will depend on `demux-eos`

If that's the intent, then I guess that's fine... 🤔